Db/index.js

```diff
@@ -1,104 +1,99 @@
-      created_by VARCHAR(40),
-      last_updated VARCHAR(7),
-      language VARCHAR(25),
-      img_url VARCHAR(100),
+      rating NUMERIC(2, 1),
+      count_ratings SERIAL,
+      enrollment SERIAL,
+      created_by NAME, 
+      created_at VARCHAR(7),
+      updated_at VARCHAR(7),
+      language VARCHAR(17),
+      img_url VARCHAR(61),
       list_price VARCHAR(7),
       discount_price VARCHAR(7),
-      video_hrs DECIMAL(3, 1),
-      total_articles INTEGER(3),
-      total_downloads INTEGER(3),
-      active_coupon VARCHAR(11)
+      video_hrs NUMERIC(3,1),
+      total_articles SERIAL, 
+      total_downloads SERIAL,
+      active_coupon VARCHAR(11),
+      cc_options VARCHAR(27)
     );`)
-    // creates CC table
     .then(() => {
-      db.queryAsync(`
-        CREATE TABLE IF NOT EXISTS CC (
-          id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
-          cc_option VARCHAR(27)
-      );`);
-    })
-    .then(() => {
-      db.queryAsync(`
-        CREATE TABLE IF NOT EXISTS Course_CC (
-          course_id INTEGER(3),
-          cc_id INTEGER(1)     
-      );`);
-    })
-    .then(() => {
-      db.queryAsync(`
-        ALTER TABLE Course_CC ADD FOREIGN KEY (course_id) REFERENCES Course (id);
-      `);
-    })
-    .then(() => {
-      db.queryAsync(`
-        ALTER TABLE Course_CC ADD FOREIGN KEY (cc_id) REFERENCES CC (id);
-      `);
+      console.log('Table Created!');
     })
+    .catch(err => console.log(err))
 );
 
 const populateCourseData = () => {
-  const { courseSeeds } = seeds;
-  const queryStr = 'INSERT INTO Course SET ?';
-  const promises = courseSeeds.map(seed => db.queryAsync(queryStr, seed));
-  return Promise.all(promises);
+  client.query(
+    `COPY course(id, title, description, tag, rating, count_ratings, enrollment, created_by, created_at, updated_at, language, img_url, list_price, discount_price, video_hrs, total_articles, total_downloads, active_coupon, cc_options) FROM '${coursesCSV}' DELIMITER ',' CSV HEADER;`,
+  )
+    .then((data) => {
+      console.log(data);
+      console.log('Courses correctly copied!');
+    })
+    .catch(err => console.log(err));
 };
 
-const populateCCData = () => {
-  const { ccSeeds } = seeds;
-  const queryStr = 'INSERT INTO CC SET ?';
-  const promises = ccSeeds.map(seed => db.queryAsync(queryStr, seed));
-  return Promise.all(promises);
-};
 
-const populateCourseCCData = () => {
-  const { courseCCSeeds } = seeds;
-  const queryStr = 'INSERT INTO Course_CC SET ?';
-  const promises = courseCCSeeds.map(seed => db.queryAsync(queryStr, seed));
-  return Promise.all(promises);
-};
+client.connect()
+  .then(() => console.log('Postgres Connected!'))
+  // .then(() => {
+  //   dropDatabase();
+  // })
+  // .then(() => {
+  //   createDatabase();
+  // })
+  .then(() => {
+    createTables();
+  })
+  .then(() => {
+    populateCourseData();
+  })
+  .catch(err => console.log(err));
+  
+// client.query('DROP DATABASE IF EXISTS headerSidebar')
```

Comment:
This large comment block can probably be deleted before you commit.

Reply:
@Kestrad Thank you for your help, Diana!